src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs

```diff
@@ -218,7 +218,7 @@ internal void FormatArgumentsIfNeeded(ILogger logger)
                 }
             }
 
-            logger.LogDebug($"Worker description arguments after formatting: {Arguments}");
+            logger.LogDebug($"Worker description arguments after formatting: {string.Join(" ", Arguments)} ");
```

Comment:
Are we not printing the arguments somewhere later (when we start the language worker process)? Is that not enough? Would this information redundant in that case?

Reply:
@shreyas-gopalakrishna

Reply:
Yes, Found the below log that gives us the arguments info. This log can be removed.
![image](https://user-images.githubusercontent.com/11889130/213310553-c1e58563-fee3-4297-ac8f-988c3c944b72.png)